pkg/util/limitrange/limitrange.go

```diff
@@ -65,7 +66,7 @@ func (s Summary) validatePodSpecContainers(containers []corev1.Container, path *
 	if !found {
 		return nil
 	}
-	reasons := []string{}
+	var reasons []string
```

Comment:
We should not distinguish between empty and null slices here.

This is proven playground: https://go.dev/play/p/3rBybY2SBiU